scylla_post_install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [ ! -d /run/systemd/system ]; then
+    exit 0
+fi
+
+# Install capabilities.conf when AmbientCapabilities supported
+. /etc/os-release
+
+# the command below will still work in systems without system (like docker) despite versioning
+SYSTEMD_VER=`( systemctl --version || echo 0 0) | head -n1 | awk '{print $2}'`
```

Comment:
s/without system/without systemctl/

What do you mean "despite versioning"?

Please use the more modern $(...) instead of backtick...backtick.